canonical/ubuntu.com
# feat: Add lsn notices to list of references

## Done

- This addresses an issue of CVE pages not having a link back to LSN's
- Loop through notice_ids and if an LSN exists, add it to the list of references
- DJLint

## QA

- Go to https://ubuntu-com-14212.demos.haus/security/notices/LSN-0104-1 (or any LSN)
- Scroll down to 'References' and click any internal link, for example https://ubuntu-com-14212.demos.haus/security/CVE-2023-6270
- Again scroll down to 'References' and see there is a link back to the original notices page, https://ubuntu-com-14212.demos.haus/security/notices/LSN-0104-1 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14292
